deb/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+die () {
+    >&2 echo "$1"
+    exit 1
+}
+
+
+(
+    rm -rf package || die "Failed to remove old packages"
+)
+
+(
+    mkdir ../build
+    cd ../build
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr .. || die "Failed to configure project"
+    make -j "$(nproc)" || die "Failed to build project"
```

Comment:
I generally assumed that all the scripts are only executed under Linux / Debian (you need something like `dpkg`, which is not available under every Linux distro). As far as I have seen, `cmake --build` is valid for Windows systems and `-DCMAKE_BUILD_TYPE=Release` for Linux systems (I have oriented myself to https://stackoverflow.com/a/67425558). I'm not that familiar with cmake myself. Does `--build` also automatically set `-DCMAKE_BUILD_TYPE=Release` on Linux systems? Which cmake would not accept `-DCMAKE_BUILD_TYPE=Release`?

Reply:
`CMAKE_BUILD_TYPE` is platform agnostic. You can set it to release on any platform and cmake will choose the build flags accordingly. In general `cmake --build` replaces the `make` command. The build is still in two steps: `cmake`, `cmake --build` and `cmake --install` eventually.
The assumption that dpkg is available on a Linux platform is reasonable. However the assumption that the build tool is `make` isn't. You can skip changing the commands to `cmake - E` but use `cmake --build` instead of `make`

Reply:
I have updated the PR, it now uses both `cmake --build` and `cmake --install`. Thanks for the hints and the nice explanation!